Lib/multiprocessing/shared_memory.py

```diff
@@ -74,6 +74,8 @@ class SharedMemory:
     def __init__(self, name=None, create=False, size=0):
         if not size >= 0:
             raise ValueError("'size' must be a positive integer")
+        if create and size == 0:
```

Comment:
Why add the test for create? And why drop the test for negative values of size?

Reply:
@ericvsmith , if size is negative, line 74 will raise `ValueError`, if line 77 is executed, it is certain that `size >= 0`.

Reply:
> Why add the test for create?

The issue mentioned in bpo only arises if create is True, it's okay to pass `size=0`, for already created shared_memory segment, since it will be ignored.

Reply:
Ah. I missed that you weren't changing line 75, but rather adding new lines. Sorry about that.